tests/tests.js

```diff
@@ -588,14 +600,6 @@ describe('unicodePropertyEscapes', () => {
 			'(?:(?:\\uD838[\\uDEC0-\\uDEF9\\uDEFF]))'
 		);
 	});
-	it('throws without the `u` flag', () => {
-		assert.throws(() => {
-			rewritePattern('\\p{ASCII_Hex_Digit}', '', features);
```

Comment:
`/\p{ASCII_Hex_Digit}/` is a valid pattern, and it matches the string `p{ASCII_Hex_Digit}`.